docs/drivers.rst

```diff
@@ -45,6 +45,10 @@ Helper Libraries
 
 These libraries build on top of the low level APIs to simplify common tasks.
 
+.. toctree::
+
+    PixelMap <https://docs.circuitpython.org/projects/pixelmap/en/latest/>
```

Comment:
Isn't this an LED helper and could go below?

Reply:
I will move it to that section. LEDs are the only current usage. I think the class itself is general enough it could have other applications where "list like" list of things that have indexes and the user wants to have arbitrary mappings of subgroups within. Bitmaps behave kind of similar to neopixel object in that regards, I was thinking it could be used with them and displayio at some point, but don't have a specific concrete idea in mind.